hazelcast/src/main/java/com/hazelcast/internal/diagnostics/MetricsPlugin.java

```diff
@@ -40,7 +40,7 @@
      * The MetricsPlugin periodically writes the contents of the MetricsRegistry
      * to the logfile. For debugging purposes make sure the minimum metrics
      * level is set to {@link com.hazelcast.internal.metrics.ProbeLevel#DEBUG}
-     * with {@link com.hazelcast.config.MetricsConfig#setMinimumLevel(ProbeLevel)}.
+     * with {@link com.hazelcast.config.MetricsConfig#setLevel(ProbeLevel)}.
```

Comment:
Link with `MetricsConfig#setLevel` leads to nowhere. I guess that this javadoc needs to be updated, so that it mentions `hazelcast.metrics.debug.enabled` property.

Reply:
Thanks, updated.